src/vm/gdbjit.cpp

```diff
@@ -1692,13 +1802,16 @@ static int getNextPrologueIndex(int from, const SymbolsInfo *lines, int nlines)
     return -1;
 }
 
-int SymbolCount = 0;
-NewArrayHolder<Elf_Symbol> SymbolNames;
-NotifyGdb::AddrSet codeAddrs;
+static NotifyGdb::AddrSet codeAddrs;
 
 /* Create ELF/DWARF debug info for jitted method */
 void NotifyGdb::MethodCompiled(MethodDesc* MethodDescPtr)
 {
+    FunctionMemberPtrArrayHolder method;
+
+    int SymbolCount = 0;
+    NewArrayHolder<Elf_Symbol> SymbolNames;
```

Comment:
Could you please be consistent with the naming convention used in this function and use camel casing for locals?